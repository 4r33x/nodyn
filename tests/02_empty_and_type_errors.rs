// empty no longer causes errors (or anything else but waist)

nodyn::wrap! {
  TestTypes {
        * mut u32,
  }
}

fn main() {}
