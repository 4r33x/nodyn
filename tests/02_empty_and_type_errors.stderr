error: This type can't be used
 --> tests/02_empty_and_type_errors.rs:5:9
  |
5 |         * mut u32,
  |         ^
